tests/end-to-end/director/eventing_test.go

```diff
@@ -0,0 +1,145 @@
+package director
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+
+	gcli "github.com/machinebox/graphql"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetDefaultRuntimeForEventingForApplication(t *testing.T) {
+	ctx := context.Background()
+	runtimeEventingURLLabelKey := "runtime/event_service_url"
+	runtime1EventingURL := "http://eventing.runtime1.local"
+	runtime2EventingURL := "http://eventing.runtime2.local"
+	defaultScenarios := []string{"DEFAULT"}
+
+	application := registerApplication(t, ctx, "app-test-eventing")
+	defer unregisterApplication(t, application.ID)
+
+	runtime1 := registerRuntime(t, ctx, "runtime-1-eventing")
+	defer unregisterRuntimeWithinTenant(t, runtime1.ID, defaultTenant)
+
+	setRuntimeLabel(t, ctx, runtime1.ID, scenariosLabel, defaultScenarios)
+	setRuntimeLabel(t, ctx, runtime1.ID, runtimeEventingURLLabelKey, runtime1EventingURL)
+
+	runtime2 := registerRuntime(t, ctx, "runtime-2-eventing")
+	defer unregisterRuntimeWithinTenant(t, runtime2.ID, defaultTenant)
+
+	setRuntimeLabel(t, ctx, runtime2.ID, scenariosLabel, defaultScenarios)
+	setRuntimeLabel(t, ctx, runtime2.ID, runtimeEventingURLLabelKey, runtime2EventingURL)
+
+	testApp := getApp(ctx, t, application.ID)
```

Comment:
We can use `getApplication` from `fixtures_queries_test.go` and remove `getApp` function totally.